jib-cli/src/main/java/com/google/cloud/tools/jib/cli/CommonContainerConfigCliOptions.java

```diff
@@ -176,4 +178,21 @@
     }
     return Optional.empty();
   }
+
+  /**
+   * Returns {@code true} if the user-specified base image is jetty or if a custom base image is not
+   * specified.
+   *
+   * @param baseImage the base image
+   * @return a boolean
+   * @throws InvalidImageReferenceException if image reference is invalid
+   */
+  public static Boolean isJetty(Optional<String> baseImage) throws InvalidImageReferenceException {
```

Comment:
What if we make this non-static and examine the `from` field of the class?

Reply:
Great catch! That makes more sense.